Empire/PageInjection.php

```diff
@@ -198,6 +222,12 @@ public function injectPixel() {
                 window.empire.apps = window.empire.apps || {};
                 window.empire.apps.ads = window.empire.apps.ads || {};
                 window.empire.apps.ads.config = window.empire.apps.ads.config || {};
+            <?php if ( $this->empire->useInjectedAdsConfig() ) { ?>
+
```

Comment:
You can probably remove this empty line

Reply:
This is intentional; with this empty line, it looks much nicer in output HTML. Without it, there will be weird indentation.

Reply:
Classic PHP 😆